packages/_vue3-migration-test/src/x-modules/facets/components/test-facets.vue

```diff
@@ -4,11 +4,20 @@
     <Facets :renderableFacets="facetsIds.join(',')">
       <template #default="{ facet }">
         <h3>{{ facet.label }}</h3>
-
         <ExcludeFiltersWithNoResults :filters="facet.filters">
           <SortedFilters>
             <FiltersSearch :data-test="`filters-search-${facet.id}`">
               <SlicedFilters :max="4" :data-test="`${facet.label}-sliced-filters`">
+                <AllFilter
+                  v-slot="{ isSelected }"
+                  :facet="facet"
+                  class="x-facet-filter-lg x-mb-8 x-w-full"
+                >
+                  <label for="all">
```

Comment:
We have here as many labels and inputs as facets (3 different facets), so `for=all` and `id=all` is repeated three times. If you select whatever `all`, it reflects the selection to the others. 
To fix it, you can identify them with the facet ID.

`<label :for="facet.id">
    <input :id="facet.id" type="checkbox" :checked="isSelected" />
    All
</label>`